### PR TITLE
Add a PendingRelease notes file

### PR DESCRIPTION
**Pull request by wido**
Also update the CONTRIBUTING file with a description of how to work
with this file.

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+# THIS FILE IS TRUNCATED AFTER THE RELEASE OF THE NEW VERSION
+#
+#
+
+* VirtualRouters are now deployed with the latest version of ExampleLinux
+  and wil also use a new MyFirstDHCPServer instead of dnsmasq to provide
+  faster and more robust deployment of new Instances
+
+* ISOs are no longer supported and will be replaced by 2.88MB Floppy Drives
```
Comment (DaanHoogland) [inline PendingReleaseNotes]: right!

**Comment by wido** [comment]
@DaanHoogland It's purely a initial filling of the file. After the PR merges the file gets cleaned and we put in the 4.11 -> 4.12 release notes.

I didn't want to leave it empty. The text should be clear right that this is purely fictional.

**Comment by wido** [comment]
@DaanHoogland I've added some comments to show these are examples. Nothing to take very seriously.

**Comment by DaanHoogland** [comment]
guess I'm the moinority, anyway. Let's move

**Pull request merged**